### ISSUE TITLE
ListView Color Display Abnormality

### ISSUE THREAD

**Issue by w065** [ysc3839/win32-darkmode#25]
When I switch the ListView to dark mode using the following code:
`SetWindowTheme(hListView, L"ItemsView", NULL); // DarkMode`

The dark mode effect is applied successfully, but all my custom colors are lost. It appears like this:
![image](https://github.com/ysc3839/win32-darkmode/assets/105098765/447dcaeb-c115-45e6-9ef2-338dbe5f57b7)
![image](https://github.com/ysc3839/win32-darkmode/assets/105098765/e91559ae-e6cb-4187-8c53-fdb58dd6629b)

**Comment by NicusorN5** [comment]
Hi, you marked this as "complete", but you didn't tell us the solution. How did you set the custom colors?

**Comment by w065** [comment]
In fact, I made a mistake. The ListView is created within a tab control, where the tab control's procedure function is subclassed when dark mode is enabled. I overlooked this detail. The correct approach is to handle the ListView's drawing messages within the subclassed function.

The final result looks like this:
![image](https://github.com/ysc3839/win32-darkmode/assets/105098765/8ead34bf-ac22-452f-ad38-d0501c529a11)
